src/module.c

```diff
@@ -292,18 +292,27 @@ static int parseCountArgument(RedisModuleCtx *ctx,
 int TSDB_info(RedisModuleCtx *ctx, RedisModuleString **argv, int argc) {
     RedisModule_AutoMemory(ctx);
 
-    if (argc != 2) {
+    if (argc < 2 || argc > 3) {
         return RedisModule_WrongArity(ctx);
     }
 
+    RedisModuleKey *key = RedisModule_OpenKey(ctx, argv[1], REDISMODULE_READ);
     Series *series;
-    RedisModuleKey *key;
-    const int status = GetSeries(ctx, argv[1], &key, &series, REDISMODULE_READ);
```

Comment:
@danni-m I believe we should keep the GetSeries wrapper. Any reason for changing back to OpenKey+KeyType+GetValue?

Reply:
@filipecosta90 I remember to have some reasoning behing it, but looking at the code I can't seem to understand why I did it. will revert back to using GetSeries